adafruit_74hc595.py

```diff
@@ -97,23 +101,29 @@ def value(self, val: bool):
             self._shift_register.gpio = gpio
 
     @property
-    def direction(self):
+    def direction(self) -> "typing.Optional[digitalio.Direction]":
```

Comment:
Since it can only output one single thing, this can't return `None` so it shouldn't be `Optional`.  I think the typing for this getter (and the corresponding setter) can just be `digitalio.Direction.OUTPUT`.